<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NavBar</class>
 <widget class="QWidget" name="NavBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>258</width>
    <height>56</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="ControlBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>258</width>
     <height>56</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {
     border: 1px solid gray;
}

QToolButton {
     background: transparent;
     min-width: 32px;
	 min-height: 32px;
     max-width: 32px;
	 max-height: 32px;
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QToolButton" name="BeginBtn">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>13</y>
      <width>34</width>
      <height>34</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>12</width>
      <height>12</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>go to first card of lexicon</string>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonIconOnly</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="PrevBtn">
    <property name="geometry">
     <rect>
      <x>53</x>
      <y>13</y>
      <width>34</width>
      <height>34</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>12</width>
      <height>12</height>
     </size>
    </property>
    <property name="toolTip">
     <string>go to previous card of lexicon</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="RtnBtn">
    <property name="geometry">
     <rect>
      <x>93</x>
      <y>13</y>
      <width>34</width>
      <height>34</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>12</width>
      <height>12</height>
     </size>
    </property>
    <property name="toolTip">
     <string>go back</string>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="NextBtn">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>13</y>
      <width>34</width>
      <height>34</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>12</width>
      <height>12</height>
     </size>
    </property>
    <property name="toolTip">
     <string>go to next card of lexicon</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="popupMode">
     <enum>QToolButton::DelayedPopup</enum>
    </property>
    <property name="arrowType">
     <enum>Qt::NoArrow</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="EndBtn">
    <property name="geometry">
     <rect>
      <x>213</x>
      <y>13</y>
      <width>34</width>
      <height>34</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>12</width>
      <height>12</height>
     </size>
    </property>
    <property name="toolTip">
     <string>go to last cd of lexicon</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="FwdBtn">
    <property name="geometry">
     <rect>
      <x>133</x>
      <y>13</y>
      <width>34</width>
      <height>34</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>12</width>
      <height>12</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>34</width>
      <height>34</height>
     </size>
    </property>
    <property name="toolTip">
     <string>go forward</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
